### ISSUE TITLE
Path of the saved file?

### ISSUE THREAD

**Issue by Nylfii** [Mincka/DMArchiver#53]
I used the tool until the number of processed tweets stopped, then pressed Ctrl + C
If I did it right, where's the file saved? I can't find it >.<

**Comment by Mincka** [comment]
Hi, problem solved in DM. But one question, did you block me? The conversation ended strangely.

**Comment by Nylfii** [comment]
Truthfully, yes. Yes I did block you. The reason is that a couple hours later I noticed that I'm only following 1 person on Twitter. Instead of the 30-40 I was following before I used your tool. I am unsure if you did that, after all I entered my login credentials in your tool, wouldn't surprise me if you actually caught them off. But that is only an assumption. I still thank you for the tool, but sadly lost trust since it is REALLY weird that it happened after I used your tool.

Have a nice day.
